prediction_prophet/functions/research.py

```diff
@@ -20,6 +24,8 @@ def research(
     model: str = "gpt-4-0125-preview",
     initial_subqueries_limit: int = 20,
     subqueries_limit: int = 4,
+    max_results_per_search: int = 5,
```

Comment:
Exposed this as an arg too so that it's clear for the user the max potential number of sites that will be scraped (i.e. `max_results_per_search` * `subqueries_limit`)